mlflow/server/js/src/experiment-tracking/pages/prompts/utils.ts

```diff
@@ -15,6 +21,19 @@ export const PROMPT_TYPE_CHAT = 'chat' as const;
 export const PROMPT_TYPE_TAG_KEY = '_mlflow_prompt_type';
 // Tag key used to store comma-separated experiment IDs associated with a prompt
 export const PROMPT_EXPERIMENT_IDS_TAG_KEY = '_mlflow_experiment_ids';
+// Tag key used to store model config as JSON string (must match backend)
+export const MLFLOW_PROMPT_MODEL_CONFIG = 'mlflow.prompt.modelConfig';
```

Comment:
Can you please check my comment in https://github.com/mlflow/mlflow/pull/19174/files#r2605798430?

Reply:
thanks for catching it, fixed!